modules/yieldlabBidAdapter.js

```diff
@@ -80,8 +80,8 @@ export const spec = {
         const bidResponse = {
           requestId: bidRequest.bidId,
           cpm: matchedBid.price / 100,
-          width: primarysize[0],
-          height: primarysize[1],
+          width: customsize[0],
+          height: customsize[1],
           creativeId: '' + matchedBid.id,
```

Comment:
Can you add a test for this change, or for the behavior in general?

Reply:
The issue happens, if there is more than one size available. I've added a testcase for this.